### ISSUE TITLE
IIAB cannot detect Ubermix using /ro if installing onto /ro itself = /dev/sda2 = "Default System" = "Base System Partition"

### ISSUE THREAD

**Issue by holta** [iiab/iiab#1383]
@jvonau do you know a better way to detect that IIAB's installing onto Ubermix, when /ro and /rw are not visible?  (i.e. when overlay partition aufs [formerly known as UnionFS] is disabled during IIAB installation[*], such that the implementer is booting directly into /ro made temporarily writable.)

(In order to fix #1380's auto-install of /etc/tmpfiles.d/iiab.conf onto Ubermix which is no longer happening!)

CLARIF: this was not an issue last week when we were installing IIAB onto /rw = /dev/sda3 = "User Changes" partition as outlined in http://www.ubermix.org/customization.html

[*] fyi @avni disables the overlay partition (aufs = /dev/sda3 = User Changes) as follows:
- hit ESC very early on during boot when the "-" appears (flashing?) in the top-left
- hit "e" to edit [in grub]
- add " aufs=off" on the end of the 3rd line from the end [the line that begins with "linux"]
- hit Ctrl-x or F10 to boot (just once!) with aufs overlay disabled
( The official instructions are Steps 2-to-6 under "Rebuilding the Base Image" at http://www.ubermix.org/customization.html )

Tangentially related: #1382

**Comment by holta** [comment]
We can trigger off the word `ubermix` in /etc/lsb-release whose 4th and final line makes this possible:
```
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=18.04
DISTRIB_CODENAME=bionic
DISTRIB_DESCRIPTION="ubermix 4.1 (Ubuntu 18.04.1 LTS)"
```
Noting that /etc/lsb-release may not exist at all, e.g. on Raspbian!

**Comment by holta** [comment]
Implemented by PR #1385
